[GTK][WPE] Gardening LayoutTests/webrtc/video-disabled-black.html

abstractmachines opened the pull request:
#### c2c3e60de05fa77c39257724d1237e9585604f2e
<pre>
[GTK][WPE] Gardening LayoutTests/webrtc/video-disabled-black.html

Unreviewed test gardening.

Test was gardened a couple of weeks ago in webkit.org/b/261024; and
webkit.org/b/254212 is also related to this test as a fix. Since
webkit.org/b/254212, test has been flaky; prior to that, it failed in bots.
Generally a flaky test as noted in 261024 which is still open.

Noted in existing bugs 261024, 254212; moved expectations up to glib.

* LayoutTests/platform/glib/TestExpectations: move expectation here for flakiness
* LayoutTests/platform/gtk/TestExpectations:

Canonical link: <a href="https://commits.webkit.org/268498@main">https://commits.webkit.org/268498@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f87fa2a6e77ace4b0289be54fc73178a8209ee49

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/19778 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/20201 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/20819 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/21673 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/18484 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/23461 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/20345 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/20037 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/19998 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/19994 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/17201 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/22528 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/17167 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/17996 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/24282 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/18242 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/18170 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/22264 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/18766 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/15900 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/17927 "Built successfully") | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/22279 "Built successfully") | | | 
| [✅ 🛠 🧪 unsafe-merge](https://ews-build.webkit.org/#/builders/22/builds/2435 "Built successfully and passed tests") | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/18592 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

clopez commented on LayoutTests/platform/glib/TestExpectations:
```diff
@@ -3470,6 +3470,7 @@ webkit.org/b/261024 svg/text/small-fonts-in-html5.html [ Failure ImageOnlyFailur
 webkit.org/b/261024 webrtc/video-maxFramerate.html [ Pass Timeout ]
 webkit.org/b/261024 webrtc/video-replace-muted-track.html [ Pass Timeout ]
 webkit.org/b/261024 webrtc/video-unmute.html [ Pass Timeout ]
+webkit.org/b/261024 webrtc/video-disabled-black.html [ Skip ]
```
We usually prefer to indicate the possible outcomes of the test (Crash Pass) rather than skipping it.

If a test is skipped then the bots will not run it, so we will not get any further notification if/when the test starts passing back or if it has a different result (like Timeout).

abstractmachines replied:
Done :) Thanks for the note. Now this just needs proper labels to merge, e.g. `unsafe-merge-queue`, would appreciate that if you'd add those labels @clopez (and also remove `merging-blocked` label if applicable). Thanks!

abstractmachines committed:
[GTK][WPE] Gardening LayoutTests/webrtc/video-disabled-black.html

Unreviewed test gardening.

Test was gardened a couple of weeks ago in webkit.org/b/261024; and
webkit.org/b/254212 is also related to this test as a fix. Since
webkit.org/b/254212, test has been flaky; prior to that, it failed in bots.
Generally a flaky test as noted in 261024 which is still open.

Noted in existing bugs 261024, 254212; moved expectations up to glib.

* LayoutTests/platform/glib/TestExpectations: move expectation here for flakiness
* LayoutTests/platform/gtk/TestExpectations:

Canonical link: https://commits.webkit.org/268498@main
